core/utils/colour.js

```diff
@@ -30,9 +31,9 @@ goog.provide('Blockly.utils.colour');
  * @return {?string} A string containing a hex representation of the colour,
  *   or null if can't be parsed.
  */
-Blockly.utils.colour.parse = function(str) {
+const parse = function(str) {
```

Comment:
You will see in team chat that I've made an inquiry about whether we prefer this style, or it might be better to write
```JS
function parse (str) {
```

Reply:
Since we resolved to keep this format, I will rebase.